platforms-wg/README.md

```diff
@@ -9,10 +9,10 @@ To register for updates and to "Join" the group, please also consider joining th
 
 ## Chairs
 
-* Josh Gavant (@joshgav)
-* Roberth Strand (@roberthstrand)
+* Technical Lead - Roberth Strand (@roberthstrand)
```

Comment:
IMO this should be a different section as TL and chair are actively different. 

(And of course for all languages 🙈 though this language seems to be missing the meeting updates)